Assets/ReverseSnake/Scripts/Helpers/CameraHelper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Assets.src;
+
+public class CameraHelper : MonoBehaviour
```

Comment:
This is not a helper. Helper is a set of simple functions (almost always pure functions). But here I see the successor of the `MonoBehavior` with some crucial logic. I think that this is a component or some another type of the class.